python/cuml/manifold/simpl_set.pyx

```diff
@@ -138,6 +139,15 @@ def fuzzy_simplicial_set(X,
     umap_params.deterministic = <bool> deterministic
     umap_params.set_op_mix_ratio = <float> set_op_mix_ratio
     umap_params.local_connectivity = <float> local_connectivity
+    if metric.lower() in metric_parsing:
+        umap_params.metric = metric_parsing[metric.lower()]
+    else:
+        raise ValueError("Invalid value for metric: {}"
+                         .format(metric))
+    if metric_kwds is None or len(metric_kwds) == 0:
+        umap_params.p = <float> 2.0
+    else:
+        umap_params.p = <float>metric_kwds.get('p')
```

Comment:
The `metric_kwds` parameter is set to a dictionary if None, so this should always work, no?
```suggestion
    umap_params.p = <float> metric_kwds.get("p", 2.0)
```